### PR TITLE
Add test to lookup builder

### PR DESCRIPTION
**Pull request by motemotech**

### CODE REVIEW

File: src/frontend/dsl/lb.rs [hunk 1]
```diff
@@ -201,3 +201,187 @@ impl<F: Clone + Debug> LookupBuilder<F> for LookupTableBuilder<F> {
...
+
+    #[test]
+    fn test_in_place_lookup_builder_build() {
+        let in_place_lookup_builder = InPlaceLookupBuilder::<Fr>::default();
+        let annotation = in_place_lookup_builder.lookup.annotation;
+        let exprs = in_place_lookup_builder.lookup.exprs;
+        let enable = in_place_lookup_builder.lookup.enable;
+        assert_eq!(annotation, "");
+        assert_eq!(format!("{:?}",exprs), "[]");
+        assert_eq!(format!("{:?}", enable), "None");
```
Comment (leolara) [inline src/frontend/dsl/lb.rs:224]: @motemotech cannot you compare to value None without converting to string?

File: src/frontend/dsl/lb.rs [hunk 2]
```diff
@@ -201,3 +201,187 @@ impl<F: Clone + Debug> LookupBuilder<F> for LookupTableBuilder<F> {
...
+    use super::*;
+
+    #[test]
+    fn test_in_place_lookup_builder_build() {
+        let in_place_lookup_builder = InPlaceLookupBuilder::<Fr>::default();
+        let annotation = in_place_lookup_builder.lookup.annotation;
+        let exprs = in_place_lookup_builder.lookup.exprs;
+        let enable = in_place_lookup_builder.lookup.enable;
+        assert_eq!(annotation, "");
+        assert_eq!(format!("{:?}",exprs), "[]");
```
Comment (leolara) [inline src/frontend/dsl/lb.rs:223]: @motemotech I think you can check exprs.len() eq 0, without converting to string

**Review by leolara — APPROVED** [review]

**Comment by leolara** [comment]
@motemotech it is approved, a few simple comparisions do not need converting to string.

**Comment by qwang98** [comment]
@motemotech Feel free to merge it after fixing the CI checks.

**Comment by leolara** [comment]
@motemotech are you going to fix the problems?

**Pull request closed without merging**